### PR TITLE
Update error message

### LINKED ISSUE #4898: Improve MagdaCatalogItem error messages
In v8, there is no `MagdaCatalogItem` but in some cases the following error message may pop out:
image

How to reproduce:
- Let magda post the following message to terria map (The message processing in terriajs is WIP atm.)
```
{
  "catalog": [
    {
      "name": "Credit Licence Dataset - National Map",
      "type": "magda",
      "recordId": "dist-dga-548075a1-c36e-4837-bf26-cc00567b5c23",
      "url": "https://dev.magda.io/"
    }
  ],
  "baseMapName": "Positron (Light)"
}
```
- Add the item in the catalog to the map
- Share the map
- Open the shared link

### PR DESCRIPTION
**Pull request by mwu2018**
### What this PR does

Fixes #4898 

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,9 +3,12 @@ Change Log
 
 ### MobX Development
 
-#### next release (8.0.0-alpha.59)
+#### next release (8.0.0-alpha.60)
```
Comment (soyarsauce) [inline CHANGES.md]: We don't need this, it's a bit confusing - the next release is .59

**Review by soyarsauce — CHANGES_REQUESTED** [review]

**Comment by soyarsauce** [comment]
Once changelog fixed we can merge!

File: CHANGES.md (hunk 1 above)
Comment (mwu2018) [reply CHANGES.md]: fixed

Comment (soyarsauce) [reply CHANGES.md]: Ok what I meant is, keep the heading `#### next release (8.0.0-alpha.59)`, and remove the heading `#### next release (8.0.0-alpha.60)` - we haven't released 8.0.0-alpha.59 yet 🙂

Comment (soyarsauce) [reply CHANGES.md]: Like so: https://github.com/TerriaJS/terriajs/pull/4918/commits/faba6347d473bfb5e6f0c37d68700d93e1673586

**Comment by soyarsauce** [comment]
[Latest 'code change' passed via ](https://github.com/TerriaJS/terriajs/pull/4918/commits/9e27992e3c8e33194ed9aa0b7b125922279865d9)

**Review by soyarsauce — APPROVED** [review]

**Comment by mwu2018** [comment]
Thanks @soyarsauce

**Pull request merged**